packages/domain-search/src/ui/domain-search-controls/filter-popover.tsx

```diff
@@ -0,0 +1,115 @@
+import {
+	Button,
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	CheckboxControl,
+	FormTokenField,
+} from '@wordpress/components';
+import { useI18n } from '@wordpress/react-i18n';
+import { FilterState } from '../../page/types';
```

Comment:
I know we're importing from `page/types` within `components/cart.tsx` but ideally this shouldn't happen. Let's invert the order, here and in cart.

Both components export their desired shape, then `page/types` consume from it.